code/modules/power/supermatter/supermatter.dm

```diff
@@ -319,28 +410,25 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 
 /obj/machinery/power/supermatter_crystal/update_overlays()
 	. = ..()
+	if(psyCoeff > 0)
```

Comment:
Shouldn't it also check for the alpha? It might not be able to clear the overlay if psyCoeff is 0 but the overlay is still there.

Reply:
I'm pretty sure old overlays are already cleared when we call the proc.

Reply:
I could not replicate the visual bugs you told me about over on discord. If you could reproduce it again please let me know. Resolving for now.